### DISCUSSION TITLE
R - Statistik - Bagaimana menginterpretasi hasil plot() pada objek model linear regresi?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#74] [Q&A] [answered]
Berdasarkan hasil penerapan fungsi plot() pada suatu objek model linear regresi, muncul 4 plot, di antaranya adalah 2 berikut ini. Mohon bantuannya untuk memahami dan menginterpretasi 2 plot berikut?

![000013](https://user-images.githubusercontent.com/95233522/165926011-92dfbc41-2a63-4a76-aa2a-701251657d75.png)

![000012](https://user-images.githubusercontent.com/95233522/165926005-e2eb1d61-cf1e-43c1-8e3e-702bec3c4626.png)

**Comment by muhrisman** [comment, marked as answer]
Plot residual dan leverage tersebut dapat diartikan apakah terdapat titik-titik data yang berpengaruh terhadap model regresi yang dihasilkan. Untuk menerjemahkan plot tersebut dapat memperhatikan titik-titik mana saja yang jatuh dari garis merah cook's distance. Berdasarkan plot di atas, dapat terlihat titik-titik hitam poin observasi jatuh diantara garis merah yang berarti titik-titik tersebut tidak menunjukan poin yang high influence. Adapun referensi untuk penerjemahan plot tersebut dapat bapak baca pada link berikut: https://www.statology.org/residuals-vs-leverage-plot/
